eng/pipelines/templates/jobs/ci.yml

```diff
@@ -189,6 +189,7 @@ jobs:
           arguments: >
             -SourceDirectory $(System.DefaultWorkingDirectory)/build
             -TargetDirectory $(Build.ArtifactStagingDirectory)
+            -Artifacts ('${{ replace(convertToJson(parameters.Artifacts), '''', '`''') }}' | ConvertFrom-Json | Where-Object -Not skipPublishPackage )
```

Comment:
I don't really like copying this in so many places but I also cannot come up with a easy way to share it so I guess we will have to live with it. I guess we could consider just passing the raw json string to the script and doing the filtering and such in there but I don't know if it will give us much.